reloj-checador/app/views/employees/index.html.erb

```diff
@@ -1,39 +1,51 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Employees</h1>
+<div class="container">
+  <h1 class="title">Employees</h1>
+  <div class="content">
+    <%= link_to 'New Employee', new_employee_path, :class => "btn btn-new" %>
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Email</th>
+          <th>Name</th>
+          <th>Lastname</th>
+          <th>Position</th>
+          <th>Private number</th>
+          <th>Active</th>
+          <th>Company</th>
+          <th colspan="3">Actions</th>
+        </tr>
+      </thead>
 
-<table>
-  <thead>
-    <tr>
-      <th>Email</th>
-      <th>Name</th>
-      <th>Lastname</th>
-      <th>Position</th>
-      <th>Private number</th>
-      <th>Active</th>
-      <th>Company</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @employees.each do |employee| %>
-      <tr>
-        <td><%= employee.email %></td>
-        <td><%= employee.name %></td>
-        <td><%= employee.lastname %></td>
-        <td><%= employee.position %></td>
-        <td><%= employee.private_number %></td>
-        <td><%= employee.active %></td>
-        <td><%= employee.company_id %></td>
-        <td><%= link_to 'Show', employee %></td>
-        <td><%= link_to 'Edit', edit_employee_path(employee) %></td>
-        <td><%= link_to 'Destroy', employee, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New Employee', new_employee_path %>
+      <tbody>
+        <% @employees.each do |employee| %>
+          <tr>
+            <td><%= employee.email %></td>
+            <td><%= employee.name %></td>
+            <td><%= employee.lastname %></td>
+            <td><%= employee.position %></td>
+            <td><%= employee.private_number %></td>
+            <td><%= employee.active %></td>
+            <td><%= employee.company_id %></td>
+            <td>
+              <%= link_to employee, :class => "btn btn-show" do %>
+                <i class="fas fa-eye"></i>show 
+              <% end %>
+            </td>
+            <td>
+              <%= link_to edit_employee_path(employee), :class => "btn btn-edit" do%>
+                <i class="fas fa-user-edit"></i>Edit
+              <% end %>
+            </td>
+            <td>
+              <%= link_to employee, method: :delete, data: { confirm: 'Are you sure?' }, :class => "btn btn-destroy" do %>
+                <i class="fas fa-trash"></i>Destroy
+              <% end %>
+            </td>
+          </tr>
+        <% end %>
```

Comment:
There is no need of changing anything here but just wanted to let you know that rails provide a mechanism for creating partial views. And with those, we are able to just pass a collection/list/array of elements, and it'll render just as you are doing it here.

Mentioning it because it's so common found partials-usage in rails projects 😄 

Rails Guides/ Rendering Collections: https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections